client/src/actions/HomeActions.ts

```diff
@@ -25,4 +26,14 @@ export interface AddCourseSections {
   sections: CourseObjectProps[];
 }
 
-export type HomeActions = Switch | AddCourse | AddCourseSections | SelectTerm;
+export interface SelectDays {
+  type: typeof SELECTDAYS;
+  days: number[];
```

Comment:
Representing days as an array of numbers without defining what the numbers mean is a bit confusing. Let's not use magic numbers here - unless we're defining what they mean somewhere else. We could define an enum such as:
{
 Monday: 0 
 Tuesday: 1
 etc.
}